### PR TITLE
Review and update Broker docs

### PR DESCRIPTION
**Pull request by CumpsD**
Went over the Swapping pages and fixed wherever something was not consistent or missing/wrong.

### CODE REVIEW

File: pages/mainnet/validator-setup/funding-and-bidding.mdx [hunk 1]
```diff
@@ -13,13 +13,13 @@ To fund your validator, head to the [auctions](https://auctions.chainflip.io/) a
...
 In order to fund your account, you should use the Auctions App. You could also interact with the Smart Contract directly, but since that's ugly we'll use the Dashboard for the purposes of this tutorial.
 
-1. Make sure you have `FLIP` in your Metamask.
-2. Go to [Chainflip Auctions page](https://auctions.chainflip.io/) > "**My Nodes**"
-3. Connect your Metamask wallet with the `FLIP`
-4. Click the button "**+ Add Node**" > You should see the "**Register new node**" modal
-5. Enter the Validator ID you got during [Generating Keys](../validator-setup/keys.md) step — your `Public Key(SS58)`— and the amount of `FLIP` you want to add funds to. Click on "**Add Funds**"
-6. Metamask will ask you to sign two transactions. The first one is a token approval and the second one transfers and add funds to your validator.
-7. Congrats! You should see the new node on "**My Nodes**" page
+1. Make sure you have `FLIP` in your crypto wallet.
```
Comment (fernandezdavid) [inline pages/mainnet/validator-setup/funding-and-bidding.mdx:16]: Worth mentioning FLIP is an ERC-20, or your "EVM crypto wallet"

File: pages/swapping/integrations/chainflip-account/funding.mdx [hunk 2]
```diff
@@ -10,11 +10,17 @@ description: Funding your Chainflip Account
...
 
-{/*
-*Coming soon*
+## Fund and redeem via Auctions application
 
-TODO: I would explain the various funding ways here: Auction dashboard, Smart Contract
-*/}
+The easiest way to fund your Chainflip account is via the [Auctions application](https://auctions.chainflip.io/).
+
+1. Make sure you have `FLIP` in your crypto wallet.
```
Comment (fernandezdavid) [inline pages/swapping/integrations/chainflip-account/funding.mdx:17]: Same remark about being FLIP the ERC-20 token

Comment (fernandezdavid) [reply pages/swapping/integrations/chainflip-account/funding.mdx:17]: And probably add a note that this was built for funding validator nodes, but works for any SC account

File: pages/swapping/integrations/running-a-broker/broker-api.mdx [hunk 3]
```diff
@@ -10,19 +10,18 @@ import { Callout } from "@/components";
...
-
 The Broker API is a standalone client that exposes _Broker_ functionality via a JSON API interface. It can be obtained from the same location as the chainflip-node, i.e. you can install it via `apt`, use the provided docker image or compile it yourself from the sources on Github.
 
 Brokers need to run the client for the API themselves, as the Broker holds keys used to sign extrinsics and collect any fees set for [Deposit Channels](../../../concepts/swaps-amm/deposit-channels-and-brokers). The API client works by pointing it to an RPC node — also run by the same Broker, ideally.
 
 ## 1. Running Broker API locally
 
-Before you do anything, you need to generate a valid [signing_key](/mainnet/validator-setup/keys#validator-keys) and [fund](/mainnet/validator-setup/funding-and-bidding#adding-funds-to-your-validator-node) the associated account.
-If you are planning to use our Docker setup, instructions on how to generate new keys and fund your account are provided in the github repo.
+Before you do anything, you need to generate a valid [signing_key](/mainnet/validator-setup/keys#validator-keys) and [fund](../chainflip-account/funding) the associated account.
```
Comment (fernandezdavid) [inline pages/swapping/integrations/running-a-broker/broker-api.mdx:19]: think its confusing now you jump to a "validator" section to create the `validator_keys` and then a couple of sections above to fund it

**Review by fernandezdavid — COMMENTED** [review]
Im missing the "Integrations > Quick Start" page or planning to do in a follow up?

File: pages/swapping/integrations/running-a-broker/broker-api.mdx (hunk 3 above)
Comment (CumpsD) [reply pages/swapping/integrations/running-a-broker/broker-api.mdx:19]: I agree, I will add the key creation part in here. The jumping was part of the old docs.

File: pages/mainnet/validator-setup/funding-and-bidding.mdx (hunk 1 above)
Comment (CumpsD) [reply pages/mainnet/validator-setup/funding-and-bidding.mdx:16]: Will do

**Comment by CumpsD** [comment]
> Im missing the "Integrations > Quick Start" page or planning to do in a follow up?

Quick Start comes in another PR

**Review by fernandezdavid — APPROVED** [review]

**Pull request merged**